packages/forma-36-react-components/src/utils/isBrowser.ts

```diff
@@ -1,3 +1,3 @@
-const isBrowser = () => typeof window !== 'undefined';
+const isBrowser = typeof window !== 'undefined';
```

Comment:
This was caught by the latest TS compiler which complains when it detects a function as the condition in `if (condition)`. We were using `isBrowser` without calling it as a function.